pkg/autoscaler/config/config.go

```diff
@@ -71,6 +71,10 @@ type Config struct {
 	// services. This can be set to 0 iff AllowZeroInitialScale is true.
 	InitialScale int32
 
+	// DefaultMaxScale is the default max scale for any revision created without an
+	// autoscaling.knative.dev/maxScale annotation
+	DefaultMaxScale int32
```

Comment:
same comment as elsewhere, most of these values are defaults that can be overridden, so I _think_ it's best if this is just `MaxScale` for consistency

```suggestion
	MaxScale int32
```